web/src/components/GridCell.js

```diff
@@ -32,7 +33,12 @@ class SingleGridCell extends React.Component {
   }
 
   cellClick = (event) => {
-    this.props.handleCellClick(event)
+    // this.props.handleCellClick(event)
```

Comment:
Is this commented code still needed? If not, we should remove it

Reply:
Will remove.